ciao-controller/api.go

```diff
@@ -347,15 +347,12 @@ func serversAction(c *controller, w http.ResponseWriter, r *http.Request) (APIRe
 	if len(servers.ServerIDs) > 0 {
 		for _, instanceID := range servers.ServerIDs {
 			// make sure the instance belongs to the tenant
-			instance, err := c.ds.GetInstance(instanceID)
+			_, err := c.ds.GetTenantInstance(tenant, instanceID)
 
```

Comment:
nit: Whilst you're editing this bit can you kill the whitespace between the line that generates the error and where it is tested.